### PR TITLE
Return to blocking waits in the workspace.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Just putting this out there to get jenkins results.

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
test vsi please

**Comment by CyrusNajmabadi** [comment]
retest prtest/win/vsi/p3 please

**Comment by CyrusNajmabadi** [comment]
@dotnet/roslyn-ide  Please view this asap.  THis is needed to restore stability on the jenkins machines.

File: src/Workspaces/Core/Portable/Workspace/Workspace_Editor.cs [hunk 1]
```diff
@@ -395,25 +395,26 @@ private ISet<DocumentId> GetProjectOpenDocuments_NoLock(ProjectId project)
             return openDocs;
         }
 
-        protected internal void OnDocumentOpened(
+        internal Task OnDocumentOpenedAsync(
             DocumentId documentId, SourceTextContainer textContainer,
```
Comment (davkean) [inline src/Workspaces/Core/Portable/Workspace/Workspace_Editor.cs]: This now becomes a blocking call now?

**Comment by davkean** [comment]
@jasonmalinowski @DustinCampbell @Pilchie Can we get some eyes on this?

File: src/Workspaces/Core/Portable/Workspace/Workspace_Editor.cs (hunk 1 above)
Comment (CyrusNajmabadi) [inline src/Workspaces/Core/Portable/Workspace/Workspace_Editor.cs]: Yes.  This is to revert the issue *temporarily* to unblock Jenkins.  We can then restore the async behavior next week once we can get it working properly.

I don't want to have people affected by this issue for the next 5 days over thanksgiving.

**Comment by jasonmalinowski** [comment]
:+1: for getting us unblocked.

**Comment by dpoeschl** [comment]
:+1:

**Comment by Pilchie** [comment]
:+1:

File: src/Workspaces/Core/Portable/Workspace/Workspace_Editor.cs (hunk 1 above)
Comment (davkean) [inline src/Workspaces/Core/Portable/Workspace/Workspace_Editor.cs]: Okay, can we file a bug to track this so that we don't lose it?

**Pull request merged**